packages/synapse-react-client/src/template_style/_form.scss

```diff
@@ -16,7 +16,7 @@
 
 .checkbox,
 .radio {
-  padding-left: 0px;
+  padding-left: 6px;
```

Comment:
https://sagebionetworks.jira.com/browse/PORTALS-2732 (not the complete fix, so leaving this bug as open)

Reply:
Would it make more sense to add margin to the table cell renderer?

Reply:
Recommend we do both.  
In general, I think the checkbox should not bump against the left side edge (it already seems to have right padding).  Adam wants this to be wider in the table cell context, so PORTALS-2732 is not fixed by this change (but it is improved).